taskt/UI/Forms/Supplement Forms/frmImageCapture.cs

```diff
@@ -41,59 +37,63 @@ private void frmImageCapture_Load(object sender, EventArgs e)
                 tabTestMode.Hide();
             }
 
-
-
-
-            this.FormBorderStyle = FormBorderStyle.None;
-            this.Location = new Point(0, 0);
-            this.WindowState = FormWindowState.Maximized;
+            FormBorderStyle = FormBorderStyle.None;
+            Location = new Point(0, 0);
+            WindowState = FormWindowState.Maximized;
 
             //Hide the Form
-            this.Hide();
+            Hide();
+
             //Create the Bitmap
-            Bitmap printscreen = new Bitmap(Screen.PrimaryScreen.Bounds.Width,
-                                     Screen.PrimaryScreen.Bounds.Height);
+            Bitmap printscreen = new Bitmap(Screen.PrimaryScreen.Bounds.Width, Screen.PrimaryScreen.Bounds.Height);
+
             //Create the Graphic Variable with screen Dimensions
             Graphics graphics = Graphics.FromImage(printscreen as Image);
+
             //Copy Image from the screen
             graphics.CopyFromScreen(0, 0, 0, 0, printscreen.Size);
+
             //Create a temporal memory stream for the image
             using (MemoryStream s = new MemoryStream())
             {
                 //save graphic variable into memory
                 printscreen.Save(s, ImageFormat.Bmp);
-                pictureBox1.Size = new System.Drawing.Size(this.Width, this.Height);
+
+                pbMainImage.Size = new Size(Width, Height);
+
                 //set the picture box with temporary stream
-                pictureBox1.Image = Image.FromStream(s);
+                pbMainImage.Image = Image.FromStream(s);
             }
             //Show Form
-            this.Show();
+            Show();
+
             //Cross Cursor
             Cursor = Cursors.Cross;
         }
 
         private void pictureBox1_MouseMove(object sender, MouseEventArgs e)
```

Comment:
I think this event name should also be changed after changing control name from `pictureBox1` to `pbMainImage`

Reply:
done